src/neuvector/values/unicorn-values.yaml

```
@@ -4,32 +4,39 @@
 # Generate certs missing from unicorn images
 autoGenerateCert: true
 
-registry: cgr.dev
-# renovate: datasource=docker depName=cgr.dev/du-uds-defenseunicorns/neuvector-controller-fips versioning=docker
-tag: "5.4.4"
+registry: quay.io
+# renovate: datasource=docker depName=quay.io/rfcurated/neuvector/controller versioning=docker
+tag: 5.4.4-jammy-scratch-fips-rfcurated
+
 manager:
   image:
-    repository: du-uds-defenseunicorns/neuvector-manager
+    repository: rfcurated/neuvector/managerr
```

```suggestion
    repository: rfcurated/neuvector/manager
```

Unless the image is just named funny 😅